lectures/10.HPC_and_parallel_compute/scripts/analyze_sequences.py

```diff
@@ -0,0 +1,58 @@
+import argparse
+import pathlib
+
+import pandas as pd
+
+
+def analyze_sequences(sequence: str) -> tuple:
```

Comment:
perhaps `analyze` could be strengthened by adding specificity? perhaps a better name would be: `count_cpg_sequences()`?

Reply:
renamed and refactored